.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  eslint:
+    name: ESLint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
+  # stylelint:
+  #   name: Stylelint
+  #   runs-on: ubuntu-18.04
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: "12.x"
+  #     - name: Setup Stylelint
+  #       run: |
+  #         npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  #         [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+  #     - name: Stylelint Report
+  #       run: npx stylelint "**/*.{css,scss}"
```

Comment:
- Enable your linters. More information about this, [HERE](https://github.com/microverseinc/linters-config).
![Screen Shot 2021-08-18 at 19 12 27](https://user-images.githubusercontent.com/22258274/129987925-576a2387-0462-4758-8b7c-b9b52df65224.png)


**You are missing a General requirement!!!!!** 
- Kindly provide information in your README file. (General information about your project, how to run it locally, how to install it, provide information about the authors, add a license).